hack/update-codegen.sh

```diff
@@ -11,6 +11,5 @@ bash ${CODEGEN_PKG}/generate-groups.sh "deepcopy,client,informer,lister" \
   github.com/palantir/k8s-spark-scheduler-lib/pkg/client \
   github.com/palantir/k8s-spark-scheduler-lib/pkg/apis \
   'sparkscheduler:v1beta1 scaler:v1alpha1' \
-  --output-base "$(dirname "${BASH_SOURCE[0]}")/../../.." \
```

Comment:
this had one too many `..` which caused the generated output to be put in the wrong place outside this repo. This is not required anyways, so just dropping it